CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ pico_sdk_init()
 
 # Add executable. Default name is the project name, version 0.1
 
-add_executable(jetkvm-dc jetkvm-dc.c driver_ina219.c)
+add_executable(jetkvm-dc jetkvm-dc.c driver_ina219.c flash_store.c)
 
 pico_set_program_name(jetkvm-dc "jetkvm-dc")
 pico_set_program_version(jetkvm-dc "0.1")
```

Comment:
bump the version ?

Reply:
I thought this would be done on release.

Reply:
Done